backend/app/views/spree/admin/menu_items/_form.html.erb

```diff
@@ -0,0 +1,97 @@
+<div data-hook="admin_menu_item_form_fields">
+  <div class="row">
+    <div class="col-12 col-md-6">
+      <div class="card mb-3">
+        <div class="card-header">
+          <h5 class="mb-0"><%= Spree.t('admin.navigation.menu_item_type') %></h5>
+        </div>
+        <div class="card-body">
+          <%= f.field_container :item_type, class: ['form-group'] do %>
+            <%= f.label :item_type, Spree.t('admin.navigation.item_type') %>
+            <%= f.select(:item_type, Spree::MenuItem::ITEM_TYPE, {include_blank: false}, { class: 'select2' }) %>
```

Comment:
```suggestion
            <%= f.select(:item_type, Spree::MenuItem::ITEM_TYPE, {include_blank: false}, { class: 'select2' }) %>
```

let's move that `Spree::MenuItem::ITEM_TYPE` from the views to controllers please